project/PATCH-RELEASES.md

```diff
@@ -0,0 +1,53 @@
+# Docker patch (bugfix) release process
+
+Patch releases (the 'Z' in vX.Y.Z) are intended to fix major issues in a
+release. Docker open source projects follow these procedures when creating a
```

Comment:
Also worth noting; initially I wanted to add this document to the docker/opensource repository (to reuse it for all open source projects). However, there are still differences between projects.

I decided to start with adding it to this repository, then work towards describing a reusable release policy for all projects.